.travis.yml

```diff
@@ -35,13 +36,27 @@ addons:
       - g++-5
       - gcc-5
 
+      - clang-5.0
+      - libc++-dev
+      - libc++abi-dev
+      - libclang-5.0-dev
+      - libclang-common-5.0-dev
+      - libclang1-5.0
+
       # pip3
       - python3-pip
 
+env:
```

Comment:
Problem with this token `HUNTER_RUN_UPLOAD: OFF`

Reply:
fixed ... finally!